aiida_kkr/workflows/imp_BdG.py

```diff
@@ -0,0 +1,414 @@
+# Workflow for impurity BdG calculation from converged normal state impurity portential and BdG host calculation
+
+from aiida.engine import WorkChain, ToContext, if_
+from aiida.orm import Dict, RemoteData, Code, CalcJobNode, WorkChainNode, Float, Bool, XyData, SinglefileData
+from aiida_kkr.workflows import kkr_imp_wc, kkr_imp_dos_wc
+from aiida_kkr.tools.find_parent import get_calc_from_remote
+from aiida_kkr.tools.common_workfunctions import test_and_get_codenode
+
+__copyright__ = (u'Copyright (c), 2022, Forschungszentrum Jülich GmbH, '
+                 'IAS-1/PGI-1, Germany. All rights reserved.')
+__license__ = 'MIT license, see LICENSE.txt file'
+__version__ = '0.1.0'
+__contributors__ = (u'David Antognini Silva, Philipp Rüßmann')
+
+
+class kkrimp_BdG_wc(WorkChain):
+    """
+    Workchain for blablabla
+    inputs::
+        :blabla: blabla
+    returns::
+        :blabla : blabla
```

Comment:
Add docstring for workchain documenting the inputs and outputs and the basic idea of what it is supposed to be doing